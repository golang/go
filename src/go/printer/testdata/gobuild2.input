// +build y

// other comment

package p

func f()

//go:build x
