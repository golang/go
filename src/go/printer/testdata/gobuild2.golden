//go:build x
// +build x

// other comment

package p

func f()
